x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_flyout/alerts_flyout.test.tsx

```diff
@@ -35,10 +61,93 @@ describe('AlertsFlyout', () => {
       await nextTick();
       wrapper.update();
     });
-    expect(wrapper.find('[data-test-subj="alertsFlyoutName"]').first().text()).toBe('one');
-    expect(wrapper.find('[data-test-subj="alertsFlyoutReason"]').first().text()).toBe('two');
+    expect(wrapper.find('h3').first().text()).toBe('Internal flyout body');
```

Comment:
Do we need a test for the case when there's no flyout?

Reply:
Yes, but it should in the alert table, I think